pkg/services/rpcsrv/server.go

```diff
@@ -446,6 +446,13 @@ func (s *Server) handleHTTPRequest(w http.ResponseWriter, httpRequest *http.Requ
 		return
 	}
 
+	if httpRequest.Method == "OPTIONS" && s.config.EnableCORSWorkaround { // Preflight CORS.
+		setCORSOriginHeaders(w.Header())
+		w.Header().Set("Access-Control-Allow-Methods", "GET, POST") // GET for websockets.
+		w.Header().Set("Access-Control-Max-Age", "31536000")
```

Comment:
A year of caching, we are quite stable :D

Reply:
OK, let's have some hours, someone might want to turn these options on and off.